services/tweetService.js

```diff
@@ -1,10 +1,28 @@
-const { Tweet, Reply, Like, User } = require('../models')
+const { Tweet, Reply, Like, User, Sequelize } = require('../models')
 
 const tweetService = {
-  getTweets: async (whereQuery = {}) => {
+  getTweets: async (whereQuery, admin) => {
+    if (admin) {
+      return await Tweet.findAll({
+        where: whereQuery,
+        attributes: { exclude: ['UserId'] },
+        include: [{ model: User, attributes: ['id', 'name', 'account', 'avatar'] }],
+        order: [['createdAt', 'DESC']]
+      })
+    }
```

Comment:
```suggestion
    if (admin) {
      return await Tweet.findAll({
        attributes: { exclude: ['UserId'] },
        include: [{ model: User, attributes: ['name', 'account', 'avatar'] }],
        order: [['createdAt', 'DESC']]
      })
    }
```
這一塊還是分離成兩個函式好了，比較容易維護。